manual/src/refman/patterns.etex

```diff
@@ -101,6 +137,14 @@ in case some value~$v$ matches @pattern_1 "|" pattern_2@, the bindings
 performed are those of @pattern_1@ when $v$ matches @pattern_1@.
 Otherwise, value~$v$ matches @pattern_2@ whose bindings are performed.
 
+\begin{caml_example}{toplevel}
+type shape =
+  Square of float | Rect of (float * float) | Circle of float
```

Comment:
Nitpick, but could it be `| Square of float`? This looks much better in the multiline version, and even here it would make it consistent with the matching below.